src/app/teacher/edit-tag/edit-tag.component.ts

```diff
@@ -99,6 +103,10 @@ export class EditTagComponent {
     }
   }
 
+  protected chooseColor(event: any): void {
```

Comment:
Is there a type for this event that is more specific? Is it MatChipSelectionChange? https://material.angular.io/components/chips/api#MatChipEditedEvent